.golangci.yml

```diff
@@ -5,14 +5,31 @@ run:
   tests: true
   # default is true. Enables skipping of directories:
   #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
-  skip-dirs-use-default: true
+  issues.exclude-dirs-use-default: true
   # Include non-test files tagged as test-only.
   # Context: https://github.com/ava-labs/avalanchego/pull/3173
 
 linters:
   disable-all: true
   enable:
     - goimports
+    - govet
+    - ineffassign
+    - misspell
+    - unconvert
+    - typecheck
+    - unused
+    # - staticcheck
+    - bidichk
+    - durationcheck
+    - copyloopvar
+    - whitespace
+    # - revive # only certain checks enabled
```

Comment:
this one has a separate commit, better take it along with the changes that fixes receiver names